models/fund.py

```diff
@@ -35,24 +35,15 @@ def add_round(self, fund_round: Round):
 
 class FundMethods:
     @staticmethod
-    def get_fund(fund_id: str = None, fund_short_name: str = None) -> Fund:
-        fund_short_name = request.args.get("fund", fund_short_name)
-        if fund_short_name:
-            url = (
-                Config.FUND_STORE_API_HOST + Config.FUND_STORE_FUND_ENDPOINT
-            ).format(fund_id=fund_short_name)
-        # TODO remove after R2W3 closes and fs-2505 is complete (make fund_short_name non-optional) # noqa
-        else:
-            url = (
-                Config.FUND_STORE_API_HOST + Config.FUND_STORE_FUND_ENDPOINT
-            ).format(fund_id=fund_id)
-
+    def get_fund(fund_short_name: str, use_short_name: bool = True) -> Fund:
```

Comment:
if the only arg is fund_short_name, why do we need the bool as well?

Reply:
I kept this is in as a future proofing thing (incase we do ever want to pass around ids aswell as short_names) but on second thought we can add this in the future fairly easily if needed so will remove! 👌🏾